content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -107,6 +111,47 @@ The max number of pods that is created within a single polling period. If there
 * **Running Job Count:** How many jobs are running.
 * **Number of the Scale:** The number of the job that is created.
 
+---
+
+```yaml
+strategy: "default"                 # Optional. Default: default. Which Scaling Strategy to use. 
```

Comment:
I'd add `scalingStrategy:` to the first line in this example, so it is more obvious what is this section about